scripts/overcloud-service-configuration-generate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euE
+set -o pipefail
+
+PARENT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${PARENT}/../functions"
+
+function main {
+    node_config_dir=("${@:1}")
+    log_info "Running overcloud service configuration generate --node-config-dir ${node_config_dir[@]}"
+    kayobe_init
+    run_kayobe overcloud service configuration generate "--node-config-dir ${@}"
```

Comment:
Has it been tested? I'd expect the quotes to make the last two arguments into one.

Reply:
Have tested properly now, you're right this did need to be changed. (I'd mistakenly just changed the comment in this branch before :facepalm:)